services/management/ia/templates_service.go

```diff
@@ -40,13 +41,19 @@ import (
 	"google.golang.org/grpc/status"
 	"gopkg.in/reform.v1"
 	"gopkg.in/yaml.v3"
+
+	"github.com/percona/pmm-managed/utils/dir"
 )
 
 const (
-	builtinTemplatesPath = "/tmp/ia1/*.yml"
-	userTemplatesPath    = "/tmp/ia2/*.yml"
+	builtinTemplatesPath = "../../../data/iatemplates/*.yml"
```

Comment:
That constant should not exist anymore – we should load templates from the generated Go file